src/daemon/Makefile.am

```diff
@@ -7,16 +7,18 @@ bin_SCRIPTS = \
     abrt-handle-upload
 
 bin_PROGRAMS = \
-    abrt-action-save-package-data \
```

Comment:
You have removed `abrt-action-save-package-data` from `bin_PROGRAMS` but you haven't added it to `libexec_PROGRAMS`